Snake/BusinessLayer/AI.cs

```diff
@@ -9,6 +9,14 @@ namespace BusinessLayer
     public class AI
     {
         private Direction currentDirection;
+        public Direction AIDirection

```

Comment:
consider next time not using a backing field and writing the property like this:
public Direction AIDirection{ get; private set;}